### PR TITLE
fix: brave support on Linux

### PR DESCRIPTION
**Pull request by 0xApotheosis**
## Description

- Address Cypress feedback from @rbeer and fix Cypress Brave support on Linux.
- Handle bad paths / no Brave installed more gracefully

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

A follow-on from https://github.com/shapeshift/web/pull/849

## Testing

Run using GUI: `yarn test:cypress`

### CODE REVIEW

**Comment by rbeer** [comment]
Tested on Debian
 - With Brave detected :heavy_check_mark: 
 - Without Brave detected :heavy_check_mark:

**Comment by 0xApotheosis** [comment]
> Tested on Debian
> 
>  - With Brave detected :heavy_check_mark: 
> 
>  - Without Brave detected :heavy_check_mark: 

Appreciate the help tonight/this morning, ser!

File: cypress/scripts/linux-brave-version.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+BRAVE_PATH=$(which brave-browser)
```
Comment (gomesalexandre) [inline cypress/scripts/linux-brave-version.sh:3]: `/usr/bin/which` might be a safer way to use `which`, since `which` is often aliased by some linux systems with a few additional flags like `--show-dot` or `--show-tilde`. Not sure how cypress handles these formats.

**Review by gomesalexandre — APPROVED** [review]

File: cypress/scripts/linux-brave-version.sh (hunk 1 above)
Comment (0xApotheosis) [reply cypress/scripts/linux-brave-version.sh:3]: Nice catch - @rbeer as a Linux user, would you have a moment to test that @gomesalexandre's suggested change still works on your machine?

Comment (0xApotheosis) [reply cypress/scripts/linux-brave-version.sh:4]: As discussed with @gomesalexandre via DM - this is an edge case.

So as not to block this from being merged, I've added a TODO comment, that can be tested by someone with Linux, in https://github.com/shapeshift/web/pull/954/commits/3496da207ee3b35c5dc0061e6d9d6c51c6074ee8.

**Review by 0xdef1cafe — APPROVED** [review]
